website/content/en/preview/concepts/scheduling.md

```diff
@@ -100,7 +102,17 @@ Take care to ensure the label domains are correct. A well known label like `karp
 | karpenter.k8s.aws/instance-gpu-memory       | 16384       | [AWS Specific] Number of mebibytes of memory on the GPU                                                                                     |
 | karpenter.k8s.aws/instance-local-nvme       | 900         | [AWS Specific] Number of gibibytes of local nvme storage on the instance                                                                    |
 
-### Node selectors
+#### User-Defined Labels
+
+By default, Karpenter scheduling will deny all user-defined labels. If you specify a `nodeSelector` or a required `nodeAffinity` using a label that is not well-known to Karpenter, it will not schedule for these pods. For Karpenter to become aware that it can schedule for these labels, you must specify the label in the Provisioner requirements with the `Exists` operator:
```

Comment:
Trying to reword a bit so it's clearer, not sure if this helps or not:

```suggestion
Karpenter is aware of several well-known labels by virtue of them being derived from the instance type. If you specify a `nodeSelector` or a required `nodeAffinity` using a label that is not known to Karpenter, it will not launch nodes with those labels and pods will remain pending. For Karpenter to become aware that it can schedule for these labels, you must specify the label in the Provisioner requirements with the `Exists` operator:
```